fullnode/src/main/java/io/coti/fullnode/services/TransactionService.java

```diff
@@ -78,17 +83,24 @@ public class TransactionService extends BaseNodeTransactionService {
     private IndexedNavigableSet<ExplorerTransactionData> explorerIndexedTransactionSet;
     private BlockingQueue<TransactionData> addressTransactionsByAttachmentQueue;
     private Map<Hash, NavigableMap<Instant, Set<Hash>>> addressToTransactionsByAttachmentMap;
+    private Map<Hash, Set<Hash>> addressToRejectedTransactionsMap;
     @Autowired
     private ResendTransactionRequestCrypto resendTransactionRequestCrypto;
+    @Autowired
+    private IEventService baseNodeEventService;
```

Comment:
change to eventService

Reply:
removed as we already have the same in the parent class